tests/test_sycl_queue.py

```diff
@@ -226,13 +226,17 @@ def test_meshgrid(device_x, device_y):
     [
         pytest.param("abs", [-1.2, 1.2]),
         pytest.param("arccos", [-0.5, 0.0, 0.5]),
+        pytest.param("arccosh", [1.5, 3.5, 5.0]),
```

Comment:
could you please extend also `tests/test_umath.py` with new hyperbolic functions